xilem_core/src/view.rs

```diff
@@ -113,9 +113,9 @@ pub trait View<State, Action, Context: ViewPathTracker, Message = DynMessage>:
 }
 
 #[derive(Copy, Clone, Debug, PartialEq, Eq)]
-/// An identifier used to differentiation between the direct children of a [View].
+/// An identifier used to differentiation between the direct children of a [`View`].
 ///
-/// These are [u64] backed identifiers, which will be added to the "view path" in
+/// These are [`u64`] backed identifiers, which will be added to the "view path" in
```

Comment:
:+1: looks like std does use things this way

Reply:
Nothing to do here!